erpc_c/infra/erpc_static_queue.hpp

```diff
@@ -42,6 +42,7 @@ class StaticQueue
         m_capacity = elementCount;
         m_head = 0;
         m_tail = 0;
+        m_storage = { { 0 } };
```

Comment:
I am afraid this initialization does not work and we should use instead:
memset(m_storage, 0, sizeof(m_storage));